utils.py

```diff
@@ -352,3 +354,28 @@ def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids):
     cocoEval.summarize()
 
     return cocoEval.stats[1], cocoEval.stats[8]
+
+
+def dense_crf(img, output_probs, compat_gaussian=3, sxy_gaussian=1, compat_bilateral=10, sxy_bilateral=1, srgb=50):
+
+    h = output_probs.shape[1]
+    w = output_probs.shape[2]
+
+    d = DenseCRF2D(w, h, 2)
+    U = unary_from_softmax(output_probs)
+    org_img = denormalize_img(img, mean=[0.0], std=[1.0]) * 255.
```

Comment:
@taraspiotr having mean/std defined here feels like magic numbers. If one changed std and mean in loaders it could end up being troublesome. That could be already the case with pretrained vgg/resnet where they have different preprocessing mean/std. I would move them all to one config. I also think moving those normalization constants from loaders is a good idea.